### PR TITLE
feat: state trie migration

### PR DESCRIPTION
**Pull request by Roznovjak**
This PR integrates the state trie migration pallet and sets the state version of the runtime to 1. This change enables lazy storage migration.

### CODE REVIEW

File: runtime/basilisk/src/system.rs [hunk 1]
```diff
@@ -468,3 +469,35 @@ impl pallet_multisig::Config for Runtime {
 	type MaxSignatories = MaxSignatories;
 	type WeightInfo = ();
 }
+
+pub struct TechCommAccounts;
+impl SortedMembers<AccountId> for TechCommAccounts {
```
Comment (mrq1911) [inline runtime/basilisk/src/system.rs:474]: please use the tech committee storage for this

File: node/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "basilisk"
-version = "11.0.0"
+version = "11.0.1"
```
Comment (mrq1911) [inline node/Cargo.toml:3]: this is at least a minor bump

**Review by mrq1911 — CHANGES_REQUESTED** [review]

**Comment by jak-pan** [comment]
Wait do we also need this on snek?

**Comment by Roznovjak** [comment]
> Wait do we also need this on snek?

AFAIK we have the issue with PoV size on Basilisk as well.

**Review by mrq1911 — APPROVED** [review]

**Pull request merged**